test/src/main/java/io/strimzi/test/k8s/KubeClusterResource.java

```diff
@@ -335,7 +335,7 @@ public String defaultNamespace() {
         return cmdClient().defaultNamespace();
     }
 
-    public KubeCmdClient cmdClient() {
+    public KubeCmdClient<?> cmdClient() {
```

Comment:
Why was this needed? Was it just some warning from IDE? Or some actual reason?

Reply:
Yep, it was adding warnings about raw types to tests I was using it in.